CLI/src/CLI.cs

```diff
@@ -131,17 +143,20 @@ private static ReturnCode runOnce2(TypeCobolConfiguration config, AbstractErrorW
                     //Delegate Event to handle diagnostics generated while loading dependencies/intrinsics
                     Server.AddError(errorWriter, diagEvent.Path, diagEvent.Diagnostic);
                 };
-                #endregion
+#endregion
 
                 depParser.CustomSymbols = Tools.APIHelpers.Helpers.LoadIntrinsic(config.Copies, config.Format, DiagnosticsErrorEvent); //Load intrinsic
+#if DOMAIN_CHECKER
```

Comment:
So this means that we really start to integrate this New Semantic Domain in the TypeCobol Framework There are 3 locations where a call to Tools.APIHelpers.Helpers.LoadIntrinsic is performed, we must take care about the context. So this must be done with issue https://github.com/TypeCobolTeam/TypeCobol/issues/1278.
My Main problem is the integration with the Language Server Workspace and so on.
So I agree, this can be moved there, but this suppose also that for instance, LSP has registered a ProgramSymbolTableBuilder. For now I will simply move the code.